server/integrationtests/sidebar_test.go

```diff
@@ -0,0 +1,54 @@
+package integrationtests
```

Comment:
Not super critical for this part, but I think is needed the permissions tests for the new api endpoints (look at the `server/integrationtests/permissions_test.go`

Reply:
As it's not critical I'll defer for now in favor of getting the feature merged in for release. I'll work on this after merging.